### PR TITLE
A few small changes to the layout + hardcoded origin

### PR DESCRIPTION
**Pull request by etnbrd**

### CODE REVIEW

File: dnd5e_card_generator/export/spell.py [hunk 1]
```diff
@@ -343,16 +343,18 @@ def reaction_condition_parts(self) -> list[str]:
...
             self.format_spell_school(self.school),
-            self.spell_properties_parts,
             self.format_header_separator(),
+            self.spell_properties_parts,
             self.spell_parts,
             self.upcasting_parts,
             self.paying_components_parts,
             self.reaction_condition_parts,
+            self.format_fill(),
+            'text | Clerc'
```
Comment (brouberol) [inline dnd5e_card_generator/export/spell.py]: Could you remove that atm, until we can fill it with the right values?
```suggestion
```

File: dnd5e_card_generator/export/spell.py [hunk 2]
```diff
@@ -216,7 +216,7 @@ def casting_range_text(self) -> str:
     def render_spell_parts_text(self, text: list[str]) -> list[str]:
         text_parts = self.fix_text_with_subparts(text)
         text_parts = self.fix_text_with_bullet_points(text_parts)
-        text_parts = [self.highlight_spell_text(part) for part in text_parts]
+        # text_parts = [self.highlight_spell_text(part) for part in text_parts]
```
Comment (brouberol) [inline dnd5e_card_generator/export/spell.py]: I wonder if we should add a flag to toggle this highlighting

File: dnd5e_card_generator/export/spell.py [hunk 3]
```diff
@@ -256,8 +256,8 @@ def upcasting_parts(self) -> list[str]:
             return []
         upcasting_text = self.shorten_upcasting_text()
         upcasting_text = self.fix_translation_mistakes(upcasting_text)
-        upcasting_text = self.highlight_spell_text(upcasting_text)
-        upcasting_text = self.highlight_extra_targets(upcasting_text)
+        # upcasting_text = self.highlight_spell_text(upcasting_text)
+        # upcasting_text = self.highlight_extra_targets(upcasting_text)
```
Comment (brouberol) [inline dnd5e_card_generator/export/spell.py]: ditto

File: dnd5e_card_generator/export/spell.py (hunk 1 above)
Comment (etnbrd) [reply dnd5e_card_generator/export/spell.py]: I removed this part, and uncommented the highlighting until we add a flag to toggle it.

**Review by brouberol — APPROVED** [review]
Nice, thanks!

**Pull request merged**